services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/BaseSpecification.groovy

```diff
@@ -1,6 +1,7 @@
 package org.openkilda.functionaltests
 
 import org.openkilda.functionaltests.extension.fixture.SetupOnce
+import org.springframework.test.context.ActiveProfiles
```

Comment:
You add import here but it is not used anywhere.

Reply:
fixed